research/src/pages/popup/popup.proposal.mdx

```diff
@@ -0,0 +1,342 @@
+---
+menu: Proposals
+name: Popup (Editor's Draft)
+path: /components/popup
+pathToResearch: /components/popup.research
+---
+
+import '../../styles/spec.css'
+
+**Editors:** Melanie Richards (Microsoft)
```

Comment:
I wouldn't mind being added, if you don't mind.

Reply:
Perfect, will do!